modules/conflict_resolver/test/conflict_resolverTest.php

```diff
@@ -171,94 +160,123 @@ function testConflictResolverWithoutPermission()
          $this->assertContains("You do not have access to this page.", $bodyText);
          $this->resetPermissions();
     }
-     /**
-     * Tests research function in resolved conflicts
-     * author: Wang Shen
+    /**
+     * Tests clear button in the form
+     * The form should refreash and the data should be gone.
      *
      * @return void
      */
-    function testSearchConflictResolved()
+    function testFiltersForUnresolvedConflicts()
     {
-        $this->markTestSkipped(
-            'Rewrite this part later'
-        );
-         $this->safeGet($this->url."/conflict_resolver/?submenu=resolved_conflicts");
-         $keywordElement = $this->webDriver->findElement(
-             WebDriverBy::Name("Question")
-         );
-         $keywordElement->sendkeys('TestTestTest');
-         //click show data button
-         $this->webDriver->findElement(WebDriverBy::ID("testShowData1"))->click();
-         $this->safeGet(
-             $this->url .
-             "/conflict_resolver/?submenu=resolved_conflicts&format=json"
-         );
-         $bodyText = $this->webDriver->getPageSource();
-         $this->assertContains("TestTestTest", $bodyText);
+        $this->safeGet($this->url . "/conflict_resolver/");
+        //testing data
+        // site = montreal
+        $this-> _testFilter(self::$ForSite, "20 rows displayed of 319", '2');
+        // Visit label = V1
+        $this-> _testFilter(self::$VisitLabel, "displayed of 581", '1');
+        $this-> _testFilter(self::$CandID, "8 rows displayed of 8", '300001');
+        $this-> _testFilter(self::$PSCID, "8 rows displayed of 8", 'MTL001');
+        $this-> _testFilter(self::$Question, "displayed of 182", 'height_inches');
+         // project = Pumpernickel
+        $this-> _testFilter(self::$Project, "3 rows displayed of 3", '2');
```

Comment:
@maltheism 
$this-> _testFilter(self::$Project, "3 rows displayed of 3", '2'); 
to 
$this-> _testFilter(self::$Project, "3 rows displayed of 3", '1');